UPDATE REGISTER.rtf

```diff
@@ -2,71 +2,89 @@ CLOVER Updates Register
-			Added example saved optimisations
+			Added “Optimisation inputs.csv”;
+			Added example saved optimisations.
 	10.	Addition of Optimisation.py (12.12.18)
-			Added to “Scripts” folder
-			Script for optimising system sizes
-	11. 	Addition of To Do list (07.01.19)
-			List of upcoming updates and known issues to solve
-	12. 	Bug fixes of Finance.py (21.01.19)
-			Fixing bug to enable calling inputs from "Finance inputs.csv"
-	13. 	Addition of GHGs.py to "Impact" folder (01.02.19)
-			Added to "Scripts" folder
-			Script for calculating GHG impact
+			Added to “Scripts” folder;
+			Script for optimising system sizes.
+	11. Addition of To Do list (07.01.19)
+			List of upcoming updates and known issues to solve.
+	12. Bug fixes of Finance.py (21.01.19)
+			Fixing bug to enable calling inputs from "Finance inputs.csv".
+	13. Addition of GHGs.py to "Impact" folder (01.02.19)
+			Added to "Scripts" folder;
+			Script for calculating GHG impact.
 	14.	Addition to “Impact” folder in Bahraich location folder (01.02.19)
-			Added “GHG inputs.csv”
+			Added “GHG inputs.csv”.
 	15.	Addition to “Impact” folder in New_Location folder (01.02.19)
-			Added “GHG inputs.csv”
-	16. 	Update to Optimisation.py (01.02.19)
-			Integrated GHG assessment into optimisation processes
-			Minor bug fixes
-	17. 	Update to Solar.py (14.03.19)
-			Minor bug fixes
-	18. 	Update to Grid.py (03.05.19)
-			Added function to change grid profile hours
+			Added “GHG inputs.csv”.
+	16. Update to Optimisation.py (01.02.19)
+			Integrated GHG assessment into optimisation processes;
+			Minor bug fixes.
+	17. Update to Solar.py (14.03.19)
+			Minor bug fixes.
+	18. Update to Grid.py (03.05.19)
+			Added function to change grid profile hours.
 	19.	Update to Energy_System.py (03.05.19)
-			Now able to simulate systems with no PV and/or storage
+			Now able to simulate systems with no PV and/or storage.
 	20.	Update to Optimisation.py (03.05.19)
-			Fixes to accomodate zero PV/storage size
-	21. 	Update to Optimisation.py (18.07.19)
-			Inclusion of GHG module to Optimisation.py
-			Bug fix for fraction of unmet demand calculation
-	22. 	Update to Solar.py (01.08.19)
-			Bug fix to solar degradation function to allow 20 year time horizon
+			Fixes to accomodate zero PV/storage size.
+	21. Update to Optimisation.py (18.07.19)
+			Inclusion of GHG module to Optimisation.py;
+			Bug fix for fraction of unmet demand calculation.
+	22. Update to Solar.py (01.08.19)
+			Bug fix to solar degradation function to allow 20 year time horizon.
 	23.	Update to Energy_System.py (17.02.20)
-			Allows different c-rates into/out of batteries
-	25. 	Update to Load.py (17.06.20)
+			Allows different c-rates into/out of batteries.
+	25. Update to Load.py (17.06.20)
 			Added function to calculate yearly load statistics when using a
-				load profile not generated using the standard methodology 
-			
+			load profile not generated using the standard methodology.
+	26. v5.0.0a1.dev1 (22.09.21)
```

Comment:
It may be worth archiving the previous updates and restarting the updates from here. I think the old ones aren't really relevant anymore. What do you think?

Reply:
Maybe..? Though it's quite nice having the history of development?